linera-service/src/database_tool.rs

```diff
@@ -165,9 +155,9 @@ async fn main() -> Result<(), anyhow::Error> {
                 .await
                 .expect("successful delete_all operation");
             if test {
-                tracing::error!("The database does exist");
+                process::exit(0);
             } else {
-                tracing::error!("The database does not exist");
+                process::exit(1);
```

Comment:
Exiting the current process from a function is not a great habit.

Perhaps less error-prone that returning several non-zero error codes, we could have two commands:
`check_that_database_exists` (succeeds if the database visibly exists)
`check_that_database_is_absent`  (succeeds if the database is visibly absent)

Reply:
Indeed, that is a way to reconcile what I mean by error messages and test that I was against.

Reply:
But I propose the shorter `check_existence` and `check_absence`.

Reply:
sounds good!